src/app/website/course-home-page/course-home-page-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { CourseHomePageComponent } from './course-home-page.component';
+
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+
+const compARoutes: Routes = [
```

Comment:
do not copy paste. you will not learn anything, if you don't write it by yourself